pkgs/native_assets_builder/test_data/native_add_v1_0_0/src/native_add.c

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+#include "native_add.h"
+
+int32_t add(int32_t a, int32_t b) {
```

Comment:
Most written piece of C code for testing Dart FFI!

Reply:
Correction, most copied piece of code 🤣